homeassistant/components/meteo_france/sensor.py

```diff
@@ -177,18 +180,37 @@ class MeteoFranceRainSensor(MeteoFranceSensor):
     @property
     def state(self):
         """Return the state."""
-        return self.coordinator.data.forecast[0]["desc"]
+        return (
+            dt_util.as_local(self.coordinator.data.next_rain_date_locale())
+            if self.coordinator.data.next_rain_date_locale() is not None
```

Comment:
Save `next_rain_date_locale()` into a var to avoid multiple compute.

:heavy_plus_sign: 

```suggestion
            if next_rain_date_locale
```

Reply:
done